docs/quick-reference/index.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: default
+title: Quick Reference
+nav_order: 4
+has_toc: false
+---
+
+# Quick Reference
+
+[1]: https://obsidian-tasks-group.github.io/obsidian-tasks/queries/filters/
+[2]: https://obsidian-tasks-group.github.io/obsidian-tasks/queries/sorting/
+[3]: https://obsidian-tasks-group.github.io/obsidian-tasks/queries/grouping/
+[4]: https://obsidian-tasks-group.github.io/obsidian-tasks/queries/layout/
+
+This table summarizes the filters and other options available inside a `tasks` block.
+
+| Presence<br>Absence                          | [Filter][1]                                                                        | [Sort][2]<br>[Group][3]                     | [Display][4]           |
```

Comment:
I think that glanceability would be improved by switching columns 1 and 2, as 1 is now quite sparsely populated.